apicurio-registry.yaml

```
@@ -19,9 +19,11 @@ environment:
       - go
       - icu
       - maven
+      - nginx
       - nodejs-20
       - npm
       - openjdk-17-default-jdk
+      - openssl
```

```suggestion
      - nodejs-20
      - npm
      - openjdk-17-default-jdk
```
`nginx` isnt being used anywhere here and `openssl` is unnecessary (see the other review below)